### PR TITLE
Added support for the Aventen S3 Sync

### PR DESCRIPTION
**Pull request by avencan**
## Description of Change
Added the Aventen S3 Sync development board

## Tests scenarios
NA

### CODE REVIEW

**Comment by avencan** [comment]
@P-R-O-C-H-Y  I have created a new pull request, and deleted the previous one, I hope this works.

Thanks

**Comment by avencan** [comment]
I also may need you to do a thorough overview, review to know if this will work, since I've implemented it quickly, and I haven't had enough time to thoroughly check it. 

E.G programming is usually via the onboard CP2102N, but I have also added some USB CDC lines, just incase the user may want to program it with the USB OTG functions, since the USB pins are exposed.

image

**Review by P-R-O-C-H-Y — APPROVED** [review]
Looks fine :) Thanks for creating new clean PR

File: boards.txt
Comment (P-R-O-C-H-Y) [inline boards.txt:1798]: .build.board should me uppercase.
```suggestion
aventen_s3_sync.build.board=AVENTEN_S3_SYNC
```

File: boards.txt
Comment (P-R-O-C-H-Y) [inline boards.txt:1797]: CI is failing, as you have mixed naming in variant.
variant in board.txt file is `aventen_s3_sync` but variant folder is `Aventen_S3_Sync`.

Please align the naming.
```suggestion
aventen_s3_sync.build.variant=Aventen_S3_Sync
```

**Review by P-R-O-C-H-Y — CHANGES_REQUESTED** [review]
@avencan PTAL on my comments

**Comment by avencan** [comment]
> @avencan PTAL on my comments

Hi

I've fixed the issues but now we have that issue on the big commit. Sorry about this

**Comment by P-R-O-C-H-Y** [comment]
> > @avencan PTAL on my comments
> 
> Hi
> 
> I've fixed the issues but now we have that issue on the big commit. Sorry about this

IDK why this is happening... But we cannot merge this PR if all boards are "changed" in that commit.
Can you please open a last one, with those changes I asked you to do? Thanks

**Comment by avencan** [comment]
> > > @avencan PTAL on my comments
> > 
> > Hi
> > 
> > I've fixed the issues but now we have that issue on the big commit. Sorry about this
> 
> IDK why this is happening... But we cannot merge this PR if all boards are "changed" in that commit.
> Can you please open a last one, with those changes I asked you to do? Thanks

Will do ASAP thanks

**Pull request closed without merging**
